docs/content/engines/hive.md

```diff
@@ -211,6 +211,18 @@ OK
 3	Paimon
 */
 
+```
+## Setting of location Location 
+In addition to the way setting location above, you can also place the location setting in TBProperties 
+to avoid Hive accessing Paimon's location through its own file system when creating tables.
+This method is effective in scenarios using Object storage,such as s3.
+
+```sql
+CREATE EXTERNAL TABLE external_test_table
+STORED BY 'org.apache.paimon.hive.PaimonStorageHandler'
+TBLPROPERTIES (
+ 'paimon_meta_location' ='s3://xxxxx/path/to/table/store/warehouse/default.db/test_table'
```

Comment:
After looking at the api again, I feel we can adjust the key.
Maybe `paimon_location`?